docker/volumes/db/webhooks.sql

```diff
@@ -25,81 +25,115 @@ BEGIN;
   CREATE INDEX supabase_functions_hooks_request_id_idx ON supabase_functions.hooks USING btree (request_id);
   CREATE INDEX supabase_functions_hooks_h_table_id_h_name_idx ON supabase_functions.hooks USING btree (hook_table_id, hook_name);
   COMMENT ON TABLE supabase_functions.hooks IS 'Supabase Functions Hooks: Audit trail for triggered hooks.';
-  CREATE FUNCTION supabase_functions.http_request()
+  CREATE OR REPLACE FUNCTION supabase_functions.http_request()
     RETURNS trigger
     LANGUAGE plpgsql
+    SECURITY DEFINER
+    SET search_path TO 'supabase_functions','net'
     AS $function$
-    DECLARE
-      request_id bigint;
-      payload jsonb;
-      url text := TG_ARGV[0]::text;
-      method text := TG_ARGV[1]::text;
-      headers jsonb DEFAULT '{}'::jsonb;
-      params jsonb DEFAULT '{}'::jsonb;
-      timeout_ms integer DEFAULT 1000;
-    BEGIN
-      IF url IS NULL OR url = 'null' THEN
-        RAISE EXCEPTION 'url argument is missing';
-      END IF;
+        DECLARE
+          local_request_id bigint;
+          payload jsonb;
+          url text := TG_ARGV[0]::text;
+          method text := TG_ARGV[1]::text;
+          headers jsonb DEFAULT '{}'::jsonb;
+          params jsonb DEFAULT '{}'::jsonb;
+          timeout_ms integer DEFAULT 1000;
```

Comment:
This is a bad default, even with 2000 (current default on pg_net) we're having trouble and I'll be changing it on https://github.com/supabase/pg_net/issues/86#issuecomment-1898428584.

I suggest leaving the default to pg_net and maybe recommend it to be higher than 5 seconds.

Reply:
I was keeping the same default that was already there. Happy to change it.

Reply:
How about no DEFAULT? Leaving it as NULL should let `pg_net` pick its default.

Reply:
That is okay with me.